colcon_core/command.py

```diff
@@ -189,7 +190,7 @@ def _main(
     colcon_logger.debug(f'Parsed command line arguments: {args}')
 
     # error: no verb provided
-    if args.verb_name is None and default_verb is None:
+    if not getattr(args, 'main', None):
```

Comment:
It took me a bit to follow this part of the change. The new code is harder to parse (har har) and a comment that main is either set by supplied verb or default if there is one could be helpful.

Reply:
Thanks, comment added in ecd8267.